### ISSUE TITLE
Changing the `Icon.ico` or `Icon.bmp` Files Do Not Update Window TItle Icon or Taskbar Icon

### ISSUE THREAD

**Issue by AristurtleDev** [MonoGame/MonoGame#8035]
## Issue Description
Changing the `Icon.ico` or the `Icon.bmp` files that are generated when creating a new project to anything else does not actually update the icon shown in the game window title or the task bar on windows.

## Steps to reproduce
1. Create a new project with `dotnet new mgdesktopgl -n MonoGameIconTest`
2. Replace the `Icon.ico` and the `Icon.bmp` with any image of your choice that isn't the default monogame logo icon
3. Run the project with `dotnet run`
   3.1 It doesn't matter the configuration, you can `dotnet run -c Release` as well
   
## Expected Result
The Icons that are placed in the project root should be reflected in the game window's title bar as well as the task bar in windows

## Actual Result
The icon used for the `.exe` executable that is created does have the custom icon, however the icon in the game window and the windows taskbar remains to be the default monogame icon.

See the attached image for reference
![image](https://github.com/MonoGame/MonoGame/assets/103014489/532d556b-3c44-4fd6-aa30-e7816544e19d)

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.1

#### What operating system are you using:
- Windows 11 Pro Education 64-bit (10.0 Build 22621)

#### What MonoGame platform are you using:

- DesktopGL

**Comment by AristurtleDev** [comment]
The default templates when creating a new project do not specify the `<LogicalName>` tag for the `<EmbeddedResource>` tag, so the icons are not being picked up when the `SDLGameWindow` is created here https://github.com/MonoGame/MonoGame/blob/b21463b419e55b4c898030fc22bee77dabb11210/MonoGame.Framework/Platform/SDL/SDLGameWindow.cs#L115


The templates should be adjusted to include the `<LogicalName>` tag like so

```xml
<ItemGroup>
  <EmbeddedResource Include="Icon.ico">
    <LogicalName>Icon.ico</LogicalName>
  </EmbeddedResource>
  <EmbeddedResource Include="Icon.bmp">
    <LogicalName>Icon.bmp</LogicalName>
  </EmbeddedResource>
</ItemGroup>
```

**Comment by AristurtleDev** [comment]
Just tested, this seems to only affect the DesktopGL projects.

**Comment by wazawoo** [comment]
Can confirm, including LogicalName as above resolves this on macOS as well

**Comment by SimonDarksideJ** [comment]
I would say this is the correct fix, however, the build is failing for some random reason, can anyone check it? @tomspilman / @CartBlanche

**Comment by AristurtleDev** [comment]
This was resolved with the merge of PR #8036
